command/node_status.go

```diff
@@ -325,7 +325,11 @@ func (c *NodeStatusCommand) formatNode(client *api.Client, node *api.Node) int {
 		c.Ui.Output(c.Colorize().Color(formatKV(basic)))
 
 		// Emit the driver info
-		c.outputNodeDriverInfo(node)
+		if c.verbose {
+			c.outputNodeDriverInfo(node)
+		} else {
+			c.outputTruncatedNodeDriverInfo(node)
```

Comment:
In the non-verbose case can we just add the output to the basic k/v list: https://github.com/hashicorp/nomad/pull/4098/files#diff-f2914d6bff3d2f22f80da63d7003d9fdR308